src/main/java/org/assertj/core/presentation/StandardRepresentation.java

```diff
@@ -289,7 +289,7 @@ protected String toStringOf(CompletableFuture<?> future) {
       Object joinResultRepresentation = joinResult instanceof CompletableFuture ? joinResult : toStringOf(joinResult);
       return concat(className, "[Completed: ", joinResultRepresentation, "]");
     } catch (CompletionException e) {
-      return concat(className, "[Failed: ", toStringOf(e.getCause()), "]");
+      return concat(className, "[Failed: ", toStringOf(e.getCause()), "]\n", toStringOf(e.getStackTrace()));
```

Comment:
we already display stack trace in error like [`ShouldBeInstance`](https://github.com/joel-costigliola/assertj-core/blob/cc178abc5871bc34525113550373ffd90e303034/src/main/java/org/assertj/core/error/ShouldBeInstance.java#L68), let's keep on doing that for this issue.
Replace `\n` by `String.format("%n")` for consistency.
We might end up with something like `String.format("%n%s", getStackTrace(e))`  ...